### DISCUSSION TITLE
关于社交频道

### DISCUSSION THREAD

**Discussion by sgfcz** [The-Run-Philosophy-Organization/run#387] [Ideas]
我感觉大家还是要少看看社交频道，包括discord、推特和红迪
信息杂而无章，很容易浪费时间，基本扫扫顶置贴也就差不多了
成体系的内容还是得关注github，已经够用
更别提充斥着满口支支支，但是自己又社达的一比的人了，看了更是蛋疼。。。。。。

**Comment by the-run-philosophy** [comment]
@sgfcz  谢谢，有条件的话欢迎定期把其他频道的精华内容整理到github来

**Reply by sgfcz** [reply]
可以
